### DISCUSSION TITLE
[mini.files] Auto bookmark for jumping back to last location?

### DISCUSSION THREAD

**Discussion by pkazmier** [nvim-mini/mini.nvim#1197] [Ideas]
What are your thoughts about automatically updating the `'` bookmark with the position right before someone jumps to a bookmark? Here is one use case. Let's say I want to copy a bunch of non-consecutive files to the same target location. With this feature, I can mark the destination directory with a bookmark, let's assume I use `d`. Then I go to the source directory, perform my yank, press `'d`, paste, then press `''` to immediately jump back, and then repeat the process but this time without ever needing to press `'d` again as I can just just use `''` to jump back and forth now.

**Comment by echasnovski** [comment]
Thanks for the suggestion!

Yes, I've considered that. I *barely* convinced myself that adding the whole built-in bookmarks was worth adding 100 lines to 'mini.files' file. And implementing this behavior would lead to even more, while it can be reproduced with deliberate `m'` prior to jumping.

**However**, I somehow missed the possibility of setting this bookmark *before every jump* (in my mind it was something more complicated like automated tracking of "latest visited directory" or something). Setting it just before performing a jump is only essentially +3 lines. I'll think about it.

**Comment by echasnovski** [comment]
This is now [implemented](https://github.com/echasnovski/mini.nvim/blob/14f81b96d850d561b6ed0c1e6715fdd724e4447f/doc/mini-files.txt#L194) on latest `main`. Thanks for describing in more detail how this "jump to latest" can be implemented.

**Reply by pkazmier** [reply]
Works great! Thanks!!
